contrib/zenith/pagestore_smgr.c

```diff
@@ -236,9 +238,9 @@ zenith_load(void)
 {
 	Assert(page_server_connstring && page_server_connstring[0]);
 
-	load_file("libpqpagestore", false);
-	if (page_server == NULL)
-		elog(ERROR, "libpqpagestore didn't initialize correctly");
+	// load_file("libpqpagestore", false);
```

Comment:
I guess this is not needed becuse this is now a contrib module that can use libpq functions directly. So I think this can be removed competely, not just commented out